test/fixtures/utilization/aws/ecs.json

```diff
@@ -0,0 +1 @@
+{"DockerId":"1234567890987654321"}
```

Comment:
Was this pulled in from the cross-agent tests?

Reply:
I don't believe there are any cross agent tests for this. I originally was using the example from the new agent spec for this, but it was long and almost everything was not even looked at, so I removed the unused keys in the hash and gave the id value a fake number.